hermes-console/static/partials/modal/editSubscription.html

```diff
@@ -170,6 +170,64 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit:</small> {{su
                 </div>
             </div>
 
+            <hr />
+            <div class="form-group">
+                <label class="col-md-3 control-label">Filter</label>
+                <div class="col-md-9">
+                    <div class="panel">
+                        <table class="table" ng-show="subscription.filters.length > 0">
```

Comment:
The table looks nice without `<td>` having the top border.
```html
<table class='table borderless'>
```

```css
.borderless td {
    border: none;
}
```

Plus the table has `margin-bottom: 20px;`, we could remove it.